### ISSUE TITLE
4.A Practice Exam

### ISSUE THREAD

**Issue by AnnMarieDavis** [Section508Coordinators/DHSA-TT-200-A#400]
## Issue Type 
- [ ] System Bug (the course doesn't function properly)
- [ ] Typo/grammar
- [ ] Incorrect content
- [X ] 508 issue (something is not accessible according to our own test process)
- [ ] Other, specify...[replace this text with your issue type description]

## Description
The Practice Exam answer sheet marks a failure for 4.A for the Data & Security button not being keyboard accessible at the top of the page. However, the button can be accessed via keyboard at the bottom of the page.

## Location
https://exams.section508testing.net/dhsv5/dhscourse/Lesson1/exams/web-practice/testPages/index.php?KeyID=129050

## Expected Behavior / Recommendation
According to the test process (step 3) if there is another way to access an interactive element on the page this would pass.
Possible solutions in order of preference:
- Remove the web page links at the bottom of the page (this requires a change to the web page, but would cause no additional failures)
-  Remove the failure from the exam answer sheet (this requires a change to the answer key)
- Make both page links inaccessible by keyboard (this requires a change to the answer key and the web page)

**Comment by Tonylayton1** [comment]
After reviewing these pages, its best to remove these failures from page 1 and 3 of the practice exam. There is another randomized 4.a failure on page 2, so 4.A will still be covered in the practice. Remove the randomization of 4.A on pages 1 and 3 so that it is always compliant on those pages.

**Comment by mikebarklage** [comment]
Updated on Dev.

I'm not 100% sure I got it right - 4.A was linked to other 4.* IDs, which complicates things. Here's what I removed:

Page 1 - 4.A randomization, answer r1_5 for 4.F (kept r5_3)
Page 3 - 4.A and 4.F randomization

Remaining links between 4.A, 4.C, and 4.F:

Page 1 - 4.C => 4.F
Page 2 - 4.A => 4.C, 4.F
Page 3 - none

**Comment by Tonylayton1** [comment]
The link between 4.C and 4.F is not in tact on page 1. I was able to get the keyboard trap issue, which should cause both 4.C and 4.F to fail for 5_1 and 5_3, but only 4.C failed.

**Comment by mikebarklage** [comment]
Found the bug, should be fixed now.

**Comment by Tonylayton1** [comment]
verified on dev

**Comment by AnnMarieDavis** [comment]
verified
